include/psa/crypto_sizes.h

```diff
@@ -964,7 +998,7 @@
  * See also #PSA_RAW_KEY_AGREEMENT_OUTPUT_SIZE(\p key_type, \p key_bits).
  */
 #define PSA_RAW_KEY_AGREEMENT_OUTPUT_MAX_SIZE   \
-    (PSA_BITS_TO_BYTES(PSA_VENDOR_ECC_MAX_CURVE_BITS))
```

Comment:
Hmm this conditions is always true.

But this is a problem as we need up to 1024 bytes in case of FFDH and up to 64 bytes for ECC.
There is no sense to use 1024 bytes buffer for key agreement in case of ECC.
But then we would have to allocate buffer dynamically.